scripts/perftune.py

```diff
@@ -1627,6 +1638,7 @@ def names():
                                                           "CPU cores out of available according to a 'cpu_mask' value."
                                                           "Default is 16",
                   type=int, default=16, dest='cores_per_irq_core')
+argp.add_argument('--tcp-mem-fraction', default=0.03, type=float, help="Fraction of total memory to allocate for TCP buffers")
```

Comment:
0.03 looks like a magic number to me, how about adding a comment to reference `resource::calculate_memory()`, to explain that seastar reserves 7% for the rest of the system, and to be on the safe side, we use a number less than 7%?

Reply:
yes